css/at-rules/container.json

```diff
@@ -65,7 +65,8 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": "18"
+                "version_added": "18",
+                "notes": "Does not work on <code>body</code>. See <a href='https://webkit.org/b/271040'>bug 271040</a>."
```

Comment:
"The document element cannot be a container." is probably a better way to word this.

Reply:
Updated the wording as per your suggestion.